kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/commands/converters/Functions.kt

```diff
@@ -766,6 +862,36 @@ public fun Arguments.numberList(
 ): MultiConverter<Long> =
     arg(displayName, description, NumberConverter(radix).toMulti(required, signatureTypeString = "numbers"))
 
+/**
+ * Create a long converter, for lists of arguments.
+ *
+ * @param required Whether command parsing should fail if no arguments could be converted.
+ *
+ * @see LongConverter
+ */
+public fun Arguments.longList(
+    displayName: String,
+    description: String,
+    required: Boolean = true,
+    radix: Int = 10
+): MultiConverter<Long> =
+    arg(displayName, description, LongConverter(radix).toMulti(required, signatureTypeString = "numbers"))
+
+/**
+ * Create an integer converter, for lists of arguments.
+ *
+ * @param required Whether command parsing should fail if no arguments could be converted.
+ *
+ * @see IntConverter
+ */
+public fun Arguments.intList(
```

Comment:
The functions in each region are alphabetically sorted, and yours should be no different. Please ensure that, within each region, the functions are sorted alphabetically — that means it'll be far, far easier to find what one is looking for.